back-end/routes/users.js

```diff
@@ -6,4 +6,11 @@ router.get('/', function(req, res, next) {
   res.send('respond with a resource');
 });
 
+router.get("/me", (req, res, next) => {
+  if (req.isAuthenticated()) {
+    return res.status(200).json({ user: req.user });
+  }
+  return res.json({user: null})
+});
```

Comment:
Nit/optional:

Couldn't the whole route just be

```
return res.status(200).json({ user: req.user });
```

without any conditionals?

If you're not authenticated, `req.user` will be `null`, right?